File: engine/base/src/main/java/client/database/data/CharNameAndIdData.java
package client.database.data;

public class CharNameAndIdData {
   private String name;

   private int id;

   private int buddyCapacity;

   public CharNameAndIdData(String name, int id) {
      super();
      this.name = name;
      this.id = id;
   }

   public CharNameAndIdData(String name, int id, int buddyCapacity) {
      this.name = name;
      this.id = id;
      this.buddyCapacity = buddyCapacity;
   }

   public String getName() {
      return name;
   }

   public int getId() {
      return id;
   }

   public int getBuddyCapacity() {
      return buddyCapacity;
   }
}

File: engine/model/src/main/scala/client/database/data/CharNameAndIdData.scala
package client.database.data

class CharNameAndIdData(private var _name: String, private var _id: Int, private var _buddyCapacity: Int) {
  def name: String = _name

  def id: Int = _id

  def buddyCapacity: Int = _buddyCapacity

  def this(name: String, id: Int) {
    this(name, id, 0)
  }
}
